package.json

```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "axios": "^1.0.0",
-    "hardhat": "2.14.0"
+    "hardhat": "2.14.0",
+    "starknet": "^4.22.0"
```

Comment:
Is this a necessary change? I feel like it isn't.

Reply:
added